features/response/ResponseContext.php

```diff
@@ -323,4 +325,64 @@ public function theResultShouldBeFalse()
     {
         Assert::assertSame($this->result, false);
     }
+
+    /**
+     * @When I call createMethod with the necessary data
+     */
+    public function iCallCreatemethodWithTheNecessaryData()
+    {
+        $this->result = $this->getIdBrokerClient()->createMethod(
+            '123',
+            '111111'
+        );
+    }
+
+    /**
+     * @When I call verifyMethod with the necessary data
+     */
+    public function iCallVerifyMethodWithTheNecessaryData()
+    {
+        try {
+            $this->result = $this->getIdBrokerClient()->verifyMethod(
+                '123',
+                '111111',
+                'dummy-method-submission'
+            );
+        } catch (Exception $e) {
+            $this->exceptionThrown = $e;
+        }
+    }
+
+    /**
+     * @When a Method rate-limit exception SHOULD have been thrown
```

Comment:
This has no impact, but this `@When` was probably supposed to be an `@Then`.

Reply:
Thanks.